instrumentation/metric/text.go

```diff
@@ -39,10 +37,13 @@ func (t *Text) Name() string {
 	return t.name
 }
 
-func (t *Text) Export() exportedMetric {
-	return textExport{
-		t.name,
-		t.value.Load().(string),
+func (t *Text) Export() interface{} {
+	value := t.value.Load().(string)
+	var x []interface{}
+	if err := json.Unmarshal([]byte(value), &x); err == nil {
```

Comment:
add a comment to explain why unmarshaling is required if this is a text type? is it to parse escape characters?